asnycSMTP.py

```diff
@@ -27,6 +26,14 @@ async def send_email(smtp, recipient_email, account_info, send_count):
         
     except Exception as e:
         print(f"Failed to send email from {account_info['email']} to {recipient_email}. Error: {e}")
+        if 'Connection lost' in str(e) or 'Server not connected' in str(e) or 'Connection expired' in str(e):
+            # 如果连接已过期，请重新连接
+            await smtp.quit()
+            wait_time = random.randint(300, 500)
+            print(f"Error: {e}, Waiting for {wait_time} seconds. Reconnect SMTP ")
+            await asyncio.sleep(wait_time)
+            smtp = await create_smtp_session(account_info)
+            await send_email(smtp, recipient_email, account_info, send_count)
```

Comment:
will this lead to infinite loop? 

for example, if for a specific destination mail, it keeps failing right? this will lead to for loop foever? 

i think maybe you can add a `num_of_retries` and pass in the function?